libs/blockscout-service-launcher/src/launcher/settings.rs

```diff
@@ -117,6 +119,30 @@ impl CorsSettings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(try_from = "String")]
+pub struct BasePath(pub String);
+
+impl TryFrom<String> for BasePath {
+    type Error = String;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        if !value.starts_with("/") {
+            return Err(format!(
+                "Invalid base path '{}': must start with '/'",
+                value
```

Comment:
Why not just trim those invalid prefixes and suffixes instead of returning errors?